include/spdk/env.h

```diff
@@ -191,6 +192,8 @@ int spdk_pci_device_cfg_write32(struct spdk_pci_device *dev, uint32_t value, uin
 
 bool spdk_pci_device_compare_addr(struct spdk_pci_device *dev, struct spdk_pci_addr *addr);
 
+int spdk_use_all_cpu_cores_for_app(app_callback cb, void *args);
```

Comment:
It would probably be best if the callback and this function returned a void pointer rather than int for maximum flexibility.